internal/cli/atlas/livemigrations/validation/create.go

```diff
@@ -170,14 +170,14 @@ func CreateBuilder() *cobra.Command {
 	cmd.Flags().BoolVar(&opts.sourceSSL, flag.LiveMigrationSourceSSL, false, usage.LiveMigrationSourceSSL)
 	cmd.Flags().StringVar(&opts.sourceCACertificatePath, flag.LiveMigrationSourceCACertificatePath, "", usage.LiveMigrationSourceCACertificatePath)
 	cmd.Flags().BoolVar(&opts.sourceManagedAuthentication, flag.LiveMigrationSourceManagedAuthentication, false, usage.LiveMigrationSourceManagedAuthentication)
-	cmd.Flags().StringVar(&opts.destinationClusterName, flag.LiveMigrationDestinationClusterName, "", usage.LiveMigrationDestinationClusterName)
-	cmd.Flags().StringVar(&opts.ProjectID, flag.LiveMigrationDestinationProjectID, "", usage.LiveMigrationDestinationProjectID)
+	cmd.Flags().StringVar(&opts.destinationClusterName, flag.ClusterName, "", usage.LiveMigrationDestinationClusterName)
+	cmd.Flags().StringVar(&opts.ProjectID, flag.ProjectID, "", usage.LiveMigrationDestinationProjectID)
```

Comment:
The spec had an issue I'm fixing it

cc @colm-quinn

Reply:
Should we not use the new `LiveMigrationDestinationClusterName` etc here, as the description is more accurate/helpful to users?

Reply:
the descriptions are changed, we are using the ProjectID instead for configuration wise and `clusterName` flag is widely used already in the cli.

There was a gap in the spec we basically documented both (copy/paste issue). The help will display the word `destination` to correctly explain it.

Reply:
Ah ok, didn't realise destination will show as well on the help. sgtm in that case, thanks!